tests/vppoutputencode.cpp

```diff
@@ -98,6 +102,11 @@ static void setEncodeParam(const SharedPtr<IVideoEncoder>& encoder,
     encoder->getParameters(VideoParamsTypeAVC, &encVideoParamsAVC);
     encVideoParamsAVC.idrInterval = encParam->idrInterval;
     encVideoParamsAVC.size = sizeof(VideoParamsAVC);
+    encVideoParamsAVC.enableCabac = encParam->enableCabac;
```

Comment:
build will break if you use old yami core library. better
1. increase api number in yami core library.
2. guard this code with 
```
#if YAMI_CHECK_API_VERSION(0, 2, 1)
......
#endif
```